src/api/nodeGrouping/types.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Linkurious SAS 2012 - 2024
+ *
+ * - Created on 2024-02-06.
+ */
+
+export interface NodeGroupingRule {
+  id: number;
+  sourceKey: string;
```

Comment:
instead of having sourcekey here, need to extend IDataSourceParams

Reply:
@aminerhanemi extending the IDataSourceParams here does not make sense since the `sourceKey` is always defined on the `NodeGroupingRule` interface

i will modify `CreateNodeGroupingRuleParams` interface to extend `IDataSourceParams`